eng/feeds/arm/tspconfig.yaml

```diff
@@ -6,6 +6,39 @@ options:
     emitter-output-dir: "{project-root}/.."
     azure-resource-provider-folder: "resource-manager"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-csharp":
+    package-dir: "{package-name}"
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
+    package-dir: "{package-dir}"
+    package-name: "{package-name}"
+    generate-test: true
+    generate-sample: true
+  "@azure-tools/typespec-java":
+    package-dir: "{package-dir}"
+    namespace: "{namespace}"
+    service-name: "{service-name}"
```

Comment:
I don't see `service-name` appears in the sample project.
https://github.com/Azure/azure-rest-api-specs/blob/24e1f5199983c795c042e0ebd67ce519927f8bd0/specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml#L36